parlai/mturk/core/setup_aws.py

```diff
@@ -523,6 +523,34 @@ def setup_relay_server_api(rds_host, task_files_to_copy, should_clean_up_after_u
 
     return html_api_endpoint_url, json_api_endpoint_url
 
+def calculate_mturk_cost(payment_opt):
+    """MTurk Pricing: https://requester.mturk.com/pricing
+    20% fee on the reward and bonus amount (if any) you pay Workers.
+    HITs with 10 or more assignments will be charged an additional 20% fee on the reward you pay Workers.
+
```

Comment:
Just found that MTurk has a more complex payment structure... Adding a new method to handle it